### ISSUE TITLE
what is the lowest loss we can achieve with this architecture?

### ISSUE THREAD

**Issue by laptopmutia** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#13]
I tried to overfitting the model but it seems so hard to achieve

I've minimize the data into 100 image  with 1 captions per image
and the lowest loss I can achieve in 500 epochs is around 0.9xxx

and what is the lowest loss you've achieve @sgrvinod ? thank you

**Comment by sgrvinod** [comment]
Hey, sorry, I've been really busy and I'll respond to your email (and here) in the next day or two.

I don't remember the loss values, especially the training loss - but I also don't recall noticing that it was stagnant or slowing down. I wouldn't think that the model couldn't overfit. 

To eliminate any errors in the code, were you able to get it to work on MSCOCO?

**Comment by laptopmutia** [comment]
the data is from flickr8k images and karpathy's json file but I cut it only 100 training image and 20 validation image here are the json in [my gist](https://gist.github.com/laptopmutia/241057e0acd785a99fe8598b964f7d19)

**Comment by guantinglin** [comment]
Hi, 

When train on coco, mine minimum is about 3.x. Is it normal? My generated captions seems very bad also...
